test/unit/testCheckAnswersSummaryJSONObjectBuilder.js

```diff
@@ -0,0 +1,148 @@
+    '        </div>\n' +
+    '    </dl>\n' +
+    '    <h2 class="heading-medium">The executors</h2>\n' +
+    '    <dl class="check-your-answers check-your-answers--long">\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">How many past and present executors are named on the will and any updates (&lsquo;codicils&rsquo;)?</dt>\n' +
+    '            <dd class="check-your-answers__answer">1</dd>\n' +
+    '        </div>\n' +
+    '<h3 class="heading-small">About you</h3>\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">First name(s)</dt>\n' +
+    '            <dd class="check-your-answers__answer">Bobby</dd>\n' +
+    '        </div>\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">Last name(s)</dt>\n' +
+    '            <dd class="check-your-answers__answer">Brown</dd>\n' +
+    '        </div>\n' +
+    '    </dl>\n' +
+    '    <h2 class="heading-medium">About the person who died</h2>\n' +
+    '    <dl class="check-your-answers check-your-answers--long">\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">First name(s)</dt>\n' +
+    '            <dd class="check-your-answers__answer">Graham</dd>\n' +
+    '        </div>\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">Last name(s)</dt>\n' +
+    '            <dd class="check-your-answers__answer">Greene</dd>\n' +
+    '        </div>\n' +
+    '    </dl>\n' +
+    '  </body>\n' +
+    '</html>\n';
+
+describe('CheckAnswersSummaryJSONObjectBuilder', function () {
+    beforeEach(() => {
+        checkAnswersSummaryJSONObjBuilder = new CheckAnswersSummaryJSONObjectBuilder();
+    });
+
+    describe('build()', () => {
+        it('should build the json object from html', (done) => {
+            const checkAnswersSummary = checkAnswersSummaryJSONObjBuilder.build(html);
+            assert.exists(checkAnswersSummary);
+            assertPropertyExistsAndIsEqualTo(checkAnswersSummary.mainParagraph,
+                'Check the information below carefully. This will form a record of your application for probate. It will also be stored as a public record, and will be able to be viewed online.');
+            assertPropertyExistsAndIsEqualTo(checkAnswersSummary.pageTitle, '\n' +
+                '        Check your answers\n' +
+                '    ');
+            assert.isArray(checkAnswersSummary.sections, 'Sections exists');
+            assert.lengthOf(checkAnswersSummary.sections, 5, 'Section array has length of 5');
+
+            const willSection = checkAnswersSummary.sections[0];
+            assertPropertyExistsAndIsEqualTo(willSection.title, '\n' +
+                '        The will\n' +
+                '    ');
```

Comment:
Why not use ES6 syntax for multiline strings?

Reply:
changed